Description: disable java checks
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>

--- a/android/build/core/main.mk
+++ b/android/build/core/main.mk
@@ -122,6 +122,7 @@
 endif
 
 
+ifneq (,)
 # Check for the correct version of java
 java_version := $(shell java -version 2>&1 | head -n 1 | grep '^java .*[ "]1\.6[\. "$$]')
 ifneq ($(shell java -version 2>&1 | grep -i openjdk),)
@@ -163,6 +164,7 @@
 $(info ************************************************************)
 $(error stop)
 endif
+endif
 
 ifeq (darwin,$(HOST_OS))
 GCC_REALPATH = $(realpath $(shell which gcc))
--- a/android/build/core/config.mk
+++ b/android/build/core/config.mk
@@ -340,6 +340,8 @@
 
 OLD_FLEX := prebuilts/misc/$(HOST_PREBUILT_TAG)/flex/flex-2.5.4a$(HOST_EXECUTABLE_SUFFIX)
 
+HOST_JDK_TOOLS_JAR :=
+ifneq (,)
 ifeq ($(HOST_OS),darwin)
 # Mac OS' screwy version of java uses a non-standard directory layout
 # and doesn't even seem to have tools.jar.  On the other hand, javac seems
@@ -359,6 +361,7 @@
 ifneq ($(filter 64-Bit, $(shell java -version 2>&1)),)
 HOST_JDK_IS_64BIT_VERSION := true
 endif
+endif
 
 # It's called md5 on Mac OS and md5sum on Linux
 ifeq ($(HOST_OS),darwin)
