--- gcc/debian/rules.defs~	2013-05-27 20:42:45.000000000 +0200
+++ gcc/debian/rules.defs	2013-05-30 18:05:34.681927644 +0200
@@ -311,6 +311,9 @@
   with_linaro_branch = yes
 endif
 
+# Android builds
+with_linaro_branch = yes
+
 # check if we're building for armel or armhf
 ifeq ($(DEB_TARGET_ARCH),armhf)
   float_abi := hard
@@ -1192,7 +1195,8 @@
 endif
 ifeq ($(distribution),Ubuntu)
   ifeq (,$(filter $(distrelease),dapper hardy jaunty karmic lucid maverick natty))
-    biarch_map += arm=arm
+    # not for Android
+    #biarch_map += arm=arm
   endif
 endif
 biarch_cpu := $(strip $(patsubst $(DEB_TARGET_GNU_CPU)=%,%, \
@@ -1316,6 +1320,10 @@
     biarchx32archs := /amd64/i386/
   endif
 endif
+# Android ...
+biarchhfarchs :=
+biarchsfarchs :=
+
 ifeq ($(distribution),Debian)
   #ifeq (,$(filter $(distrelease),etch squeeze wheezy))
   ifeq (,$(filter $(distrelease),etch squeeze))
